testing/travis-fork-check.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
```

Comment:
are you using `logging` because it prints to stderr? It might be simpler to just `sys.stderr.write('blah\n')`